Documents/Installation-with-Conan.md

```diff
@@ -0,0 +1,57 @@
+## Guide to Install and Setup nmos-cpp using Conan Package Manager: 
+
+1. Install conan:
+`pip install --upgrade conan>=2.0.5`
+
+    If conan.exe is not in PATH you will get warning: 
+	 
+
+	> WARNING: The script conan.exe is installed in 'C:\Users\%USER%\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\Scripts' which is not on PATH.
+	  Consider adding this directory to PATH or, if you prefer to suppress this warning, use --no-warn-script-location.
+
+
+   Then the shown path needs to be added manually. After maunally adding the PATH, Restart the command line and run `conan --help` to ensure Conan is added to path successfully.
+
+3. Detect Profile:
+	Run `conan profile detect`. \
+  This will create a conan profile with values based on your dev setup. Ideally the profile should be similar to these:
```

Comment:
- isn't this 2 not 3?
- consistent capitalization, why "Profile" capitalized here, but Conan not in "Install conan"?

~~~suggestion
2. Detect profile:
   ```sh
   conan profile detect
   ```
   This will create a Conan profile with values based on your current platform, compiler, etc.
   Ideally the profile should be similar to these:
~~~